src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenExprLambdaTests.cs

```diff
@@ -3762,19 +3762,16 @@ public void BadExprTreeLambdaInNSDeclWithCSharp6()
 namespace global::((System.Linq.Expressions.Expression<System.Func<B>>)(() => B )).Compile()(){}
 ";
             CreateCompilation(source, parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp6)).VerifyDiagnostics(
```

Comment:
note: this test is basically increidbly syntactically broken code.  we no longer get the tuple availability message because existing syntax checks prevent us from getting that far.  we have *other* existing tests that still ensure we get teh version check on tuples otherwise.